content/news/XXXX-XX-XX-mixxx-accepted-for-gsoc-2022.md

```diff
@@ -0,0 +1,13 @@
+title: Mixxx accepted for GSoC 2022!
+authors: Daniel Schürmann
+tags: gsoc, gsoc-2022
+status: draft
+comments: no
+
+Mixxx has been [accepted as a mentoring organization](https://summerofcode.withgoogle.com/programs/2022/organizations/mixxx) for [Google Summer of Code 2022](https://summerofcode.withgoogle.com/), a global program focused on bringing new contributors into open source software development.
+
+This year the program is open for all new or beginner contributors of Mixxx. It is no longer limited to enrolled students. New contributors have the opportunity to work on Mixxx full-time as a 12+ weeks programming project and receive a stipend sponsored by Google. For more information, check out the [Google Summer of Code Contributor Guide](https://google.github.io/gsocguides/student/).
+
+You will find the [GSoC 2022 Project Ideas for Mixxx](https://github.com/mixxxdj/mixxx/wiki/gsoc-2022-ideas) in our wiki, but we're also interested in project ideas that matters more to you. Taking a look at our [GSoC advice page](https://github.com/mixxxdj/mixxx/wiki/gsocadvice) is recommended if you're planning to apply. The best way to get started is to participate in the Mixxx community, so have a look at our [GitHub repository](https://github.com/mixxxdj/mixxx) and consider making a small code contribution to show that you're able to familiarize yourself with our codebase.
+
+The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```

Comment:
I'm sorry I proposed a malformed link. Can you amend and force push? 
```suggestion
The official application period will start **April 4**. Potential GSoC contributors should start now to discuss application ideas on our [Zulip](https://mixxx.zulipchat.com/). Just start a new topic in the [#gsoc stream](https://mixxx.zulipchat.com/#narrow/stream/109215-gsoc).
```

Reply:
Yes, no problem.